### PR TITLE
Add locale meta tag with simple locale information

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #253 
| License       | MIT

Would just need to convert the locale information into RFC1766 standart language codes. But have no idiea how to do that, atm. The idea was to use something from `Intl::getLocaleBundle()`.

### CODE REVIEW

File: Tests/WebTest/SeoFrontendTest.php [hunk 1]
```diff
@@ -185,4 +185,21 @@ public function testErrorHandlingForExcludedPath()
...
+        $this->assertEquals(200, $res->getStatusCode());
+
+        //test the meta tag entry
+        $metaCrawler = $crawler->filter('head > meta')->reduce(function (Crawler $node) {
+            return 'language' === $node->attr('http-equiv');
+        });
+
+        $actualMeta = $metaCrawler->extract('content', 'content');
+        $actualMeta = reset($actualMeta);
+        $this->assertEquals('en', $actualMeta);
```
Comment (ElectricMaxxx) [inline Tests/WebTest/SeoFrontendTest.php]: This should be `en_GB` or `en_US`, i think.

Comment (dantleech) [inline Tests/WebTest/SeoFrontendTest.php]: `en` is valid, the subsequent parts are optional.

Comment (dantleech) [inline Tests/WebTest/SeoFrontendTest.php]: See: http://www.langtag.net/whatare.html for example.

Comment (ElectricMaxxx) [inline Tests/WebTest/SeoFrontendTest.php]: Ok. Then this PR is ready for review.

**Comment by ElectricMaxxx** [comment]
time to merge?

**Comment by dbu** [comment]
i am still in holidays for another 10 days. if it looks good to you go
ahead and merge...

**Comment by ElectricMaxxx** [comment]
@dbu no worries. I think @lsmith77 is doing a round over the bundles everyday and merges the good ones. :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+* **2015-09-08: Add meta tag for language information
```
Comment (dbu) [inline CHANGELOG.md]: missing closing `**` after the date

Comment (dbu) [inline CHANGELOG.md]: will fix this in master

**Comment by dbu** [comment]
:+1:

**Comment by ElectricMaxxx** [comment]
thanks

**Pull request merged**